tests/ImageSharp.Benchmarks/Codecs/EncodeWebp.cs

```diff
@@ -54,16 +69,31 @@ public void ImageSharpWebpLossy()
             using var memoryStream = new MemoryStream();
             this.webp.Save(memoryStream, new WebpEncoder()
             {
-                FileFormat = WebpFileFormatType.Lossy
+                FileFormat = WebpFileFormatType.Lossy,
+                Method = WebpEncodingMethod.Level4,
+                UseAlphaCompression = false,
+                FilterStrength = 60,
+                SpatialNoiseShaping = 50,
+                EntropyPasses = 1
             });
         }
 
         [Benchmark(Baseline = true, Description = "Magick Webp Lossless")]
         public void MagickWebpLossless()
         {
             using var memoryStream = new MemoryStream();
-            this.webpMagick.Settings.SetDefine(MagickFormat.WebP, "lossless", true);
-            this.webpMagick.Write(memoryStream, MagickFormat.WebP);
+            var defines = new WebPWriteDefines
+            {
+                Lossless = true,
+                Method = 4,
+
+                // 100 means off.
+                NearLossless = 100
+            };
+
+            this.webpMagick.Settings.SetDefine(MagickFormat.WebP, "exact", false);
```

Comment:
@dlemstra the `exact` and `quality` parameters seem to be missing in `WebPWriteDefines`. Should they maybe be added to `WebPWriteDefines`? If you think that makes sense, i could open a PR at Magick.Net.

Reply:
Not sure if we need to `exact` option. You can use `this.webpMagick.Settings.Quality` to set the quality. Setting the quality too `100` might enable this?

Reply:
If i dont set it, the libwebp default will be use, right? The default is false.

Reply:
The `IMagickSettings` interface does not seem to have a Quality Property.

Reply:
Sorry my bad, that should be `webpMagick.Quality` instead. And I just added the `Exact` property to the `WebPWriteDefines`. This will become available in the next release.

Reply:
> Sorry my bad, that should be `webpMagick.Quality` instead. And I just added the `Exact` property to the `WebPWriteDefines`. This will become available in the next release. And setting the `Quality` to `100` will make it lossless.

No problem, thanks for adding `exact` to `WebPWriteDefines`